File: pc前端代码/melhat_pc-main/backups/visual-upgrade-20260906-143125/src/api/tts.js
import request from '@/utils/request'

// 分页查询广播记录
export function listTtsRecord(query) {
  return request({
    url: '/hat/tts/broadcast/page',
    method: 'get',
    params: query
  })
}

// 创建单播
export function addUnicast(data) {
  return request({
    url: '/hat/tts/broadcast/single-broadcast',
    method: 'post',
    data: data
  })
}

// 创建组播
export function addMulticast(data) {
  return request({
    url: '/hat/tts/broadcast/team-broadcast',
    method: 'post',
    data: data
  })
}

// 创建群播
export function addGroupcast(data) {
  return request({
    url: '/hat/tts/broadcast/group-broadcast',
    method: 'post',
    data: data
  })
}

File: 新版安卓端开发/网页展示/展示源码/lib/api/tts.dart
import 'package:rolling_intelligence_headband/http/index.dart';
import 'package:rolling_intelligence_headband/http/request_options.dart';

class TtsApi {
  /// 创建单播
  static Future<void> createSingleBroadcast({
    required String content,
    required String recipient,
  }) async {
    await http.request(
      ReqOptions(
        path: '/hat/tts/broadcast/single-broadcast',
        method: 'POST',
        params: {'content': content, 'recipient': recipient},
      ),
    );
  }

  /// 创建群播
  static Future<void> createGroupBroadcast({
    required String content,
    required String recipients,
  }) async {
    await http.request(
      ReqOptions(
        path: '/hat/tts/broadcast/group-broadcast',
        method: 'POST',
        params: {'content': content, 'recipients': recipients},
      ),
    );
  }
}
